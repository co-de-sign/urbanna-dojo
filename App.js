import React, { Component } from 'react'
import { StyleSheet, Text, View } from 'react-native'

import Header from './components/Header'
import Content from './components/Content'

import colors from './helpers/colors'

export default class App extends Component {
  render() {
    return (
      <View style={styles.container}>
        <Header />
        <Content />
      </View>
    )
  }
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: colors.background,
  },
})
