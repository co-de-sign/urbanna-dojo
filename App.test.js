import React from 'react';
import App from './App';

import renderer from 'react-test-renderer';

describe('App', () => {
  let app

  beforeEach(() => {
    rendered = renderer.create(<App />)
  })

  it('renders correctly', () => {
    expect(rendered).toMatchSnapshot()
  })
})
