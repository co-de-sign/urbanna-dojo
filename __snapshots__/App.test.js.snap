// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#FFEE99",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#385B79",
        "paddingBottom": 10,
        "paddingHorizontal": 10,
        "paddingTop": 54,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#fff",
          "fontSize": 24,
          "textAlign": "center",
        }
      }
    >
      Urbanna Beer
    </Text>
  </View>
  <RCTScrollView
    contentContainerStyle={
      Object {
        "alignItems": "center",
        "paddingBottom": 32,
      }
    }
  >
    <View>
      <View
        style={
          Object {
            "backgroundColor": "#385B79",
            "elevation": 1,
            "marginTop": 32,
            "minHeight": 224,
            "padding": 24,
            "width": 256,
          }
        }
      />
    </View>
  </RCTScrollView>
</View>
`;
